src/pudl/analysis/timeseries_cleaning.py

```diff
@@ -1529,6 +1539,22 @@ def filter_missing_values(
     return df
 
 
+@dataclass
+class ImputeTimeseriesSettings:
```

Comment:
Added this class to collect settings that we can configure per timeseries imputation. There's some other parameters in the flagging that could be added here, but I haven't messed with those settings at all, so I've left them for now.